internal/controller/instaslice_daemonset.go

```diff
@@ -435,6 +435,10 @@ func (r *InstaSliceDaemonsetReconciler) discoverMigEnabledGpuWithSlices() ([]str
 		os.Exit(1)
 	}
 	log.Info("classical resources obtained are ", "cpu", cpu, "memory", memory)
+	if instaslice == nil {
+		err := fmt.Errorf("unable to get instaslice object")
```

Comment:
should this message be unable to get an empty instaslice object?

Reply:
There is no instaslice object at all and hence the error message should be apt I thought